test.js

```diff
@@ -135,4 +135,29 @@ describe('notify(opts)', () => {
 			''
 		].join('\n'));
 	});
+
+	it('should exclude -g argument when `isGlobal` option is `false`', () => {
+		function Control() {
+			this.packageName = 'update-notifier-tester';
+			this.update = {
+				current: '0.0.2',
+				latest: '1.0.0'
+			};
+		}
+		util.inherits(Control, updateNotifier.UpdateNotifier);
```

Comment:
One step at a time! I'll make the refactor and perhaps do the class conversion as a separate PR. I don't wanna mix up to many changes at once!